### DISCUSSION TITLE
pycurl http code 206 issue

### DISCUSSION THREAD

**Discussion by Agaulke2025** [curl/curl#11432] [Q&A] [answered]
Hell there, I started playing around with getinfo() and HTTP_CODE and I am getting the http code 206 which is the code for a partial download. However I'm getting this when I quit my program, then rerun pycurl using RESUME_FROM and the whole download is successful, but its returning this http code for partial download. Is this normal with resume from even when the download completes and if so is there a way to distinguish between when a download is actually partial and when it was rerun and successful? Thanks for the help!

**Comment by dfandrich** [comment, marked as answer]
That's normal if you're downloading a range, which is (presumably) what pycurl
does with RESUME_FROM. You can see this from the command-line tool as well,
e.g.  curl -v -C 1 https://curl.se/info

**Comment by Agaulke2025** [comment]
is there a way then to distinguish between when a download is actually partial and when it was rerun and successful?

**Reply by bagder** [reply]
206 *is* successful ... A successful partial retrieve.

**Reply by Agaulke2025** [reply]
oh ok I understand, thanks for the help!
